.github/workflows/publish-v4.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    tags:
+      - 'v4\.[0-9]+\.[0-9]+'
```

Comment:
I think we could use it for tag publish. So `'[0-9]+.[0-9]+\.[0-9]+'` and the uses decides which major version